HousingFinanceInterimApi/V1/Infrastructure/DatabaseContext.cs

```diff
@@ -50,6 +50,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<AdjustmentAux>().Property(x => x.Timestamp).HasDefaultValueSql("GETDATE()");
             modelBuilder.Entity<MAProperty>().HasKey(x => x.PropRef);
             modelBuilder.Entity<UHProperty>().HasKey(x => x.PropRef);
+            modelBuilder.Entity<SSMiniTransaction>()
+                .HasKey(ss => new { ss.PropRef, ss.TagRef, ss.Origin, ss.PostDate });
```

Comment:
I tried with HasNoKey but EF doesn't let me track the entity with that. I needed this in order to do things like adding them and checking the results in the tests

Reply:
With HasNoKey:
```
System.InvalidOperationException
Unable to track an instance of type 'SSMiniTransaction' because it does not have a primary key. Only entity types with primary keys may be tracked.
```

Reply:
I suppose .. 9 col composite key it is...
https://github.com/LBHackney-IT/HFSDatabaseObjects/blob/main/stored_procedures/usp_LoadTransactionsCharges.sql#L170-L183

Reply:
https://github.com/LBHackney-IT/housing-finance-interim-api/pull/153/commits/56d7148669c7f1e4346a6c1429a99b811252b147